fcrepo-kernel/src/main/java/org/fcrepo/kernel/observer/FedoraEvent.java

```diff
@@ -184,6 +192,13 @@ public long getDate() {
         }
     }
 
+    /**
+     * Get the event ID.
+    **/
+    public String getEventID() {
```

Comment:
...however, if `getIdentifier()` is never actually used, maybe we get rid of it?